erpnext/config/desktop.py

```diff
@@ -17,6 +17,15 @@ def get_data():
 			"description": "Dive into the basics for your organisation's needs.",
 			"onboard_present": 1
 		},
+		{
+			"module_name": "Reports",
+			"category": "Modules",
+			"label": _("Reports"),
+			"color": "#3498db",
+			"icon": "octicon octicon-repo",
+			"type": "module",
+			"description": "Accounts, billing, payments, cost center and budgeting."
+		},
```

Comment:
- do not add the reports module before accounting.
- put it at the end of the existing modules